pentasharp/ViewModel/TaxiReservation/TaxiReservationViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.ViewModel.TaxiModels
+{
+    /// <summary>
+    /// Represents the view model for creating a new taxi reservation.
+    /// </summary>
+    public class TaxiReservationViewModel
```

Comment:
move this viewmodel and the others in this folder to the Request folder you'll create based on this comment https://github.com/starlabspro-internship/pentasharp/pull/24/files#r1867290098